src/Message/CIMResponse.php

```diff
@@ -7,6 +7,11 @@
  */
 class CIMResponse extends CIMAbstractResponse
 {
+    /**
+     * For Error codes: @see https://developer.authorize.net/api/reference/responseCodes.html
+     */
+    const ERROR_RESPONSE_CODE_CANNOT_ISSUE_CREDIT = 54;
```

Comment:
I think you should keep this as a string since other error codes are in string format. Keeps things consistent and you don't have to do typecasting in your code.